types/mailchimp__mailchimp_marketing/mailchimp__mailchimp_marketing-tests.ts

```diff
@@ -0,0 +1,3 @@
+// test: Usage of UMD Globals should be prohibited inside modules
+
+export {}; // dummy export to mark this file as a module
```

Comment:
this is for declaration file usage, while this is proper `.ts` file.
This file should test your declarations